### PR TITLE
Update PT strings

### PR DESCRIPTION
**Pull request by nuno**
Update PT strings

### CODE REVIEW

**Comment by hansemannn** [comment]
Can you clean-up the commits? We have 40 commits here.

**Comment by hansemannn** [comment]
P.S.: Awesome that you keep track of the localizations, thanks Nuno!

**Review by hansemannn — CHANGES_REQUESTED** [review]
Please address the previous comments to continue.

**Comment by nuno** [comment]
Not sure what happen

File: patch.diff [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/app/i18n/pt/strings.xml b/app/i18n/pt/strings.xml
+index 7dafbd5..6207fa7 100644
+--- a/app/i18n/pt/strings.xml
++++ b/app/i18n/pt/strings.xml
```
Comment (hansemannn) [inline patch.diff:4]: Not sure what this is, but looks like something with your git client went really wrong. The relevant files have not been changed, but there is a `patch.diff` here. The other changes seem to come from an outdated master-fork. Can I commit them manually or do you want to do a new PR?

**Comment by nuno** [comment]
let me try fix

File: patch.diff (hunk 1 above)
Comment (nuno) [reply patch.diff:4]: let me try fix

**Pull request closed without merging**